_dev/docker/circleci/Dockerfile

```diff
@@ -2,30 +2,18 @@ FROM circleci/node:12-stretch-browsers
 
 USER root
 RUN apt-get update && apt-get install -y \
-    bison \
-    build-essential \
-    cmake \
-    flex \
     graphicsmagick \
-    libboost-dev \
-    libboost-filesystem-dev \
-    libboost-regex-dev \
-    libboost-system-dev \
```

Comment:
I remove the deps for `fxa-amplitude-send` since it isn't in the normal test flow and these are quite large deps